cmd/api/routers/routers.go

```diff
@@ -204,6 +204,80 @@ func (c *Controller) GetLogURL(context *gin.Context) {
 	context.Set("jsonPath", jsonPath)
 }
 
+func (c *Controller) GetResourceByUid(context *gin.Context) {
```

Comment:
A costmetic thing but if we are using `GetLogURLByUid` not that URL is capitalized, maybe `UID` should be too:


```suggestion
func (c *Controller) GetResourceByUID(context *gin.Context) {
```